Sources/ForageSDK/Foundation/ForageSDK+ForageServices.swift

```diff
@@ -42,9 +42,8 @@ protocol ForageSDKService: AnyObject {
         bearerToken: String,
         merchantAccount: String,
         paymentMethodReference: String,
-        cardNumberToken: String,
         foragePinTextEdit: ForagePINTextField,
-        completion: @escaping (Result<Data?, Error>) -> Void)
+        completion: @escaping (Result<BalanceModel, Error>) -> Void)
```

Comment:
I like the use of making these types stricter, like here^ (e.g. `BalanceModel` > `Data`)!

is it safe to no longer have these types marked as optional?

Reply:
also loving the use of the suffix `Model`^

Reply:
It should be fine, the return type was never optional in the first place.